Tests/Tests.UWP/Tests.UWP.csproj

```diff
@@ -130,6 +130,18 @@
       <Link>realm-wrappers.dll</Link>
       <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
     </Content>
+    <Content Include="$(SolutionDir)wrappers\build\WindowsStore\$(Configuration)-$(RealmWrappersPlatform)\zlib1.dll" Condition="'$(RealmNoSync)' != 'true'">
```

Comment:
Do we need to make these changes to the Tests.Win32 project as well?

Reply:
Nope, actually this is no longer required and I should delete it.

Reply:
What about the references to Realm.Sync and Tests.Sync.Shared?

Reply:
Those are fine, I believe. Do you see sometging wrong?

Reply:
I meant - shouldn't we add them to the Win32 project? Seems like only the UWP one has been updated in this PR.

Reply:
Huh, I swear I added them, but maybe I forgot to push. I’ll check.